telebot/types.py

```diff
@@ -4209,6 +4214,7 @@ def to_dict(self):
             json_dict['is_flexible'] = self.is_flexible 
         return json_dict
 
+InputMessageContent = Union[InputTextMessageContent, InputLocationMessageContent, InputVenueMessageContent, InputContactMessageContent, InputInvoiceMessageContent]
```

Comment:
One notable change:
Since InputXXXMessageContent do not have a common class, I decided to declare this